### ISSUE TITLE
UI glitch with nested tab routes

### ISSUE THREAD

**Issue by gabfeudo** [Milad-Akarie/auto_route_library#1026]
This is my AutoRoute setup:

```
@MaterialAutoRouter(
  replaceInRouteName: "Page,Route",
  routes: [
    AutoRoute(path: "/login", initial: true, page: Login),
    AutoRoute(path: "/signup", page: Signup),
    AutoRoute(path: "/password-recovery", page: PasswordRecovery),
    CustomRoute(
      path: "/home",
      guards: [AuthGuard],
      page: NavigationWrapper,
      transitionsBuilder: TransitionsBuilders.noTransition,
      children: [
        AutoRoute(path: "", name: "HomepageRouter", page: Homepage),
        AutoRoute(
          path: "evaluate",
          name: "EvaluateRouter",
          page: EmptyRouterPage,
          children: [
            AutoRoute(path: "", page: Evaluate),
            AutoRoute(path: "evaluationResult", page: EvaluationResult),
          ],
        ),
        AutoRoute(path: "search", name: "SearchRouter", page: Search),
        AutoRoute(path: "profile", name: "ProfileRouter", page: Profile),
        RedirectRoute(path: "*", redirectTo: ""),
      ],
    ),
  ],
)
class $AppRouter {}
```

And in the `StatefulWidget` called `NavigationWrapper` (look at "/home" route) I setup the `AutoTabsRouter`.

The problem is that when I add children to any route inside "/home" (in this case I'm adding it to "/home/evaluate") I'm experiencing this kind of UI "glitch".

It looks like two layers overlapped but with a left padding.

This glitch is present both on “/home/evaluate” and “/home/evaluationResult”.

**NOTE**: look at the left portion of the screen, where I highlighted with the red rectangle. I had to cover the UI with black rectangles because this is a project for a client.

![7hQC4](https://user-images.githubusercontent.com/40292753/162166092-01d42243-c0b0-4e21-a154-6e29c20d2eb5.png)

Flutter doctor:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.4, on macOS 12.3.1 21E258 darwin-x64, locale
    it-IT)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.66.0)
[✓] Connected device (1 available)
[✓] HTTP Host Availability

• No issues found!
```

**Comment by Milad-Akarie** [comment]
@gabfeudo unfortunately this's a native navigator issue, hopefully the flutter team will do something about it
https://stackoverflow.com/questions/68986632/rid-of-elevation-of-nested-flutter-navigator-2-0

**Comment by gabfeudo** [comment]
Since this was a project for a client I had to find a workaround. 

The problem was the **Padding** widget. In particular, all nested routes should not be shown inside a **Padding** widget. They need to have Padding inside the route itself instead

I solved the problem removing the Padding widget inside the NavigationWrapper and putting it inside Evaluate (look at my route definition to better understand what I'm referring to
